README.md

```diff
@@ -1,5 +1,7 @@
 # Keda Manager
```

Comment:
I thought the documentation in `.docs` should relate to the Keda Module and Keda Manager is a part of the module. 
```suggestion
# Keda Module
```